polaris-cli/src/commands/polaris/migrate.ts

```diff
@@ -0,0 +1,43 @@
+import {Command, Flags} from '@oclif/core';
+import {run as runMigrator} from '@shopify/polaris-migrator';
+
+export default class Migrate extends Command {
```

Comment:
I'd recommend inheriting from the class exported by `@shopify/cli-kit`. [Here's](https://github.com/Shopify/cli/blob/main/packages/app/src/cli/commands/app/dev.ts#L6) an example.

Reply:
Looks like it's having a hard time finding the types for this import. Any suggestions?

<img width="1144" alt="Screen Shot 2022-09-14 at 12 48 16 PM" src="https://user-images.githubusercontent.com/11774595/190238029-811ac22c-2627-467e-97b4-738a92130902.png">